src/components/core/DpTreeList/DpTreeListNode.vue

```diff
@@ -53,17 +53,18 @@
         v-else
         class="min-width-25" />
     </div>
-    <draggable
-      v-model="tree"
-      v-bind="options.draggable"
+    <dp-draggable
+      :drag-across-branches="options.dragAcrossBranches"
       class="list-style-none u-mb-0 u-1-of-1"
-      :disabled="hasDraggableChildren === false"
```

Comment:
I couldn't find any case in which it would be necessary, but since it also works and I may have forgotten a case, I changed it in [fb1eab7](https://github.com/demos-europe/demosplan-ui/pull/53/commits/fb1eab79f4aa682b238c25be74f67160e10c5542).